docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.4"
 
 services:
   postgresql:
-    image: docker.io/library/postgres:12-alpine
```

Comment:
This is a major breaking change, do we want to include it here and now?

Reply:
Well, we'll need Postgres 14 or newer as 12 defaults to md5 for hashes or something, so maybe we should make fips mode a flag and not always try to enable it on startup

Reply:
#9727 for upgrade instructions